### PR TITLE
[WIP] Revised view

### PR DESCRIPTION
**Pull request by brianmcmichael**
WIP related to https://github.com/OSC/ood-activejobs/issues/107

Branched off of https://github.com/OSC/ood-activejobs/pull/88

### CODE REVIEW

**Comment by brianmcmichael** [comment]
* View has been updated to handle the key/value pattern described in #107
* Extended view is now rendered server-side and sent to the client as html.

updview

* I haven't put much thought yet into the order or content of the extended data attributes, this will need further thought and will vary depending on the attributes available from the adapter.
* I added some attributes for slurm systems, they are untested at this point.
* When a user clicks to get more data on a job, if the status has changed the row label will update.
* Removed a bunch of unused javascript.

Attributes are printed in the order they are added to the array, (see https://github.com/OSC/ood-activejobs/blob/revised_view/app/models/jobstatusdata.rb#L57-L69) add, remove, or alter the data by changing this array.

File: app/assets/javascripts/application.js.erb [hunk 1]
```diff
@@ -143,7 +143,7 @@ $(document).ready( function() {
             // Open this row
             row.child( format(row.data()) ).show();
             tr.addClass('shown');
-            $.getJSON(APP_PATH + 'json?pbsid='+row.data().pbsid+'&cluster='+row.data().cluster , function(data) {
+            $.getJSON(APP_PATH + '/json?pbsid='+row.data().pbsid+'&cluster='+row.data().cluster , function(data) {
```
Comment (brianmcmichael) [inline app/assets/javascripts/application.js.erb]: This will set the path to `../app//json?..`. It works, but it's kinda ugly.

**Comment by nickjer** [comment]
For Slurm:

image

is not necessary. This does not display the submit arguments. It displays the batch script run instead, and can be `null` if piped in through stdin. It may be best to remove it completely.

image

`Priority` may confuse users as it is more of a sys admin option than a user option. There is typically no documentation on this for users other than for sys admins.

image

Is this intended? Repeating the job id and name in both the heading and info table?

Also, you have all the info displayed in the original row (`Id`, `Name`, `User`, `Account`, `Time used`, and `Status`) except for `Queue` and `Cluster`. Maybe we should put those in the extended info as well.

**Comment by nickjer** [comment]
Clicking "Open in File Manager" takes me to:

```
https://ondemand.chpc.utah.edu/pun/sys/files/fs/uufs/chpc.utah.edu/common/home/u6012649/ondemand/dev/ood-myjobs/data/projects/default/
```

instead of 

```
https://ondemand.chpc.utah.edu/pun/sys/files/fs/uufs/chpc.utah.edu/common/home/u6012649/ondemand/dev/ood-myjobs/data/projects/default/4/
```

**Comment by nickjer** [comment]
Fixed those issues above for the Slurm view.

**Pull request merged**